### PR TITLE
Changing text and image file names from 2020 to 2021

### PR DESCRIPTION
**Pull request by fpierfed**
We are getting ready to transition the ep2020 site to ep2021 and that requires, at a minimum changing the dates :-) This PR does that.

### CODE REVIEW

File: conference/invoicing.py [hunk 1]
```diff
@@ -51,12 +51,24 @@
...
 Europython Society
 Ramnebacken 45
 424 38 Agnesberg
 Sweden
 Org ID: 802417-7704
+Contact Email: billing@europython.eu
+https://www.europython-society.org/
+""".strip()
+
+EPS_22 = """
```
Comment (malemburg) [inline conference/invoicing.py:67]: For EP2022, we will have to use the Irish VAT address and ID on the invoices. It's probably a good idea to add this in a comment.

File: p3/templatetags/meta.py [hunk 2]
```diff
@@ -16,12 +16,12 @@ def render_meta(context, title=""):
 
     description = ""
     page_title = (
-        f"{title}EuroPython 2020 Online &middot; 23-26 July 2020"
+        f"{title}EuroPython 2021 Online &middot; July 26 - Aug. 1, 2021"
```
Comment (malemburg) [inline p3/templatetags/meta.py:19]: This should read "Jul 26 - Aug 1, 2021" to be consistent with our other mentions of the dates (corresponds to Wikipedia date format standards).

File: templates/conference/header/_homepage_jumbotron.html [hunk 3]
```diff
@@ -2,14 +2,14 @@
 
 <div class="jumbotron">
     <div class="container text-center">
-        <img class="img-fluid" src="{% static 'img/ep2020-online-logo.png' %}"
-             alt="EuroPython 2020; July 23-26th 2020; Online"
-             title="EuroPython 2020; July 23-26th 2020; Online">
+        <img class="img-fluid" src="{% static 'img/ep2021-online-logo.png' %}"
+             alt="EuroPython 2021; July 26 - Aug. 1, 2021; Online"
```
Comment (malemburg) [inline templates/conference/header/_homepage_jumbotron.html]: Same as above for all mentions of the date range.

**Review by malemburg — APPROVED** [review]
Only a few nits. Looks good otherwise.

File: templates/conference/header/_homepage_jumbotron.html (hunk 3 above)
Comment (fpierfed) [reply templates/conference/header/_homepage_jumbotron.html]: Fixed

File: p3/templatetags/meta.py (hunk 2 above)
Comment (fpierfed) [reply p3/templatetags/meta.py:19]: Fixed

File: conference/invoicing.py (hunk 1 above)
Comment (fpierfed) [reply conference/invoicing.py:67]: Added

**Pull request merged**